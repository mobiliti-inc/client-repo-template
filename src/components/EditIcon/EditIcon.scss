@import "~variables";

.edit-icon {
    display: flex;
    width: 50px;
    height: 50px;
    background: $blue;
    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
    justify-content: center;
    align-items: center;
    border-radius: 50%;

    & > img {
        width: 25px;
        height: 25px;
    }
}