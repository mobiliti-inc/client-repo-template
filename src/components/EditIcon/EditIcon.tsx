import styles from "./EditIcon.scss";
import editImage from "../../assets/images/edit_image.png";

const EditIcon = () => (
	<div styleName="edit-icon">
		<img src={editImage} alt="edit icon" />
	</div>
);

export default CSSModules(EditIcon, styles, { allowMultiple: true });
