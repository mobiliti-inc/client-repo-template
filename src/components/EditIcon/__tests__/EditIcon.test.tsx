import EditIcon from '../EditIcon';


describe('EditIcon test', () => {
	it('renders', () => {
		const tree = shallow(<EditIcon />);
		matchSnapshot(tree);
	});
});
