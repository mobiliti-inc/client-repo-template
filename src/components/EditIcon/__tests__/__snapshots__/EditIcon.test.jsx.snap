// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditIcon test renders 1`] = `
<div
  className="edit-icon"
>
  <img
    alt="edit icon"
    src="test-file-stub"
  />
</div>
`;
